site/build.sbt

```diff
@@ -27,6 +27,8 @@ lazy val exerciseSettings: Seq[Setting[_]] =
   )
 
 lazy val clients = Seq(client)
+lazy val doobieVersion = "0.2.3"
+lazy val scalazVersion = "7.1.4"
```

Comment:
The "7.1.4" version is used, both for `scalaz-core` and `scalaz-concurrent`, ignoring the transitive dependency for doobie.